adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/BrokerHelper.cs

```diff
@@ -210,6 +210,11 @@ public void AcquireToken(IDictionary<string, string> brokerPayload)
                         RequestContext.Logger.ErrorPii(e);
                     }
                 }
+                else
```

Comment:
we were not doing anything with a null intent here. Maybe this exception is too specific? Could just be null intent as an error message, but one needs to check ADAL logs to know the redirect uri was incorrect. thoughts?

Reply:
agree.

Reply:
Yes this exception is to specific here, it might be any exception - not necessary related to incorrect URL,

---
In reply to: [203584606](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1140#discussion_r203584606) [](ancestors = 203584606)

Reply:
👎

Reply:
does broker throw exceptions or allow us in any way to get the details of the error condition?

---
In reply to: [205907511](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1140#discussion_r205907511) [](ancestors = 205907511,203584606)

Reply:
@SomkaPe Thanks for the feedback. I was concerned about that. will make the changes you suggested. 🐱

Reply:
done.